lib/config/config.go

```diff
@@ -325,6 +328,10 @@ func convertV14V15(cfg *Configuration) {
 	cfg.Version = 15
 }
 
+func convertV17V18(cfg *Configuration) {
+	cfg.Version = 18
+}
```

Comment:
This will probably need a magic struct that maps the same xml element into a string and then perform substitution somehow, without breaking the Unmarshal on the custom type.